src/pythoninlua.c

```diff
@@ -184,23 +184,54 @@ static int py_object_call(lua_State *L)
         ret = py_convert(L, value);
         Py_DECREF(value);
     } else {
-        char s_err[1024];
-        memset(s_err, 0, 1024);
+        char s_exc[1024];
```

Comment:
In modern C, I think we can write `char s_exc[1024] = {0}`, then we won't need the `memset`.

Reply:
Agreed, old habits :-)

Reply:
Have pushed one more commit resolving the initialization

Reply:
Looks good!